src/client/runner/automation/playback/type/type-char.js

```diff
@@ -83,6 +85,13 @@ function _typeCharToContentEditable (element, text) {
     var startNode        = currentSelection.startPos.node;
     var endNode          = currentSelection.endPos.node;
 
+    // NOTE: some browsers raises 'input' event after the element
+    // content changes, but in the others we should do it manually.
```

Comment:
@VasilyStrelyaev please, check this comment
`element content changes` => `element content is changed`